setup.py

```diff
@@ -17,7 +17,7 @@ def package_data(pkg, roots):
 
 setup(
     name='done-xblock',
-    version='0.1.0-rg',
+    version='0.1.2-rg',
```

Comment:
Why we choose from 0 to 2?

Reply:
it was missed on last provided version, so current on sga is 0.1.1 according to tag

Reply:
resolved